numpy/lib/ufunclike.py

```diff
@@ -55,6 +55,10 @@ def func(x, out=None, **kwargs):
     return func
 
 
+if not ENABLE_ARRAY_FUNCTION:
+    _fix_out_named_y = _deprecate_out_named_y
```

Comment:
Out of scope, but - It's been three releases since this was deprecated, and it's unlikely anyone is passing kwargs here anyway - perhaps we can just remove this decorator.